nativescript-angular/router/page-router-outlet.ts

```diff
@@ -149,16 +150,17 @@ export class PageRouterOutlet implements OnDestroy { // tslint:disable-line:dire
         private parentContexts: ChildrenOutletContexts,
         private location: ViewContainerRef,
         @Attribute("name") name: string,
+        @Attribute("isNSEmptyOutlet") isNSEmptyOutlet: boolean,
```

Comment:
The property can be named `isEmptyOutlet` -> it is already inside a  `PageRouterOutlet` class that is NS-specific.
Ale you can inject and define it in the same time 
```suggestion
        @Attribute("isNSEmptyOutlet") private isEmptyOutlet: boolean,
```